code/script.js

```diff
@@ -1,47 +1,219 @@
+    <option value="positive">Yes</option>
+    <option value="why">What's in it for you?</option>
+  </select>
+  `  
+  }
+  
+  //listen for which option in the dropdown menu has been chosen and trigger the right user message depending on users choice
+  document.getElementById("select").addEventListener('change', userReply3Yes)
+  document.getElementById("select").addEventListener('change', userReply3No)
+}
+
+const userReply3Yes = () => {
+  const dropdown = document.getElementById("select")                    //creating variabel for the dropdown
+  const selectanswer = dropdown.options[dropdown.selectedIndex].value;  //function that stores the option that user has chosen
+
+  //messages shown depending on users chosen option in dropdown menu
+  if (selectanswer === 'quarantine') {
+    showMessage("Quarantine... Heard about that?", 'user')
+    setTimeout(botReply4Quarantine, 1000)
+} else if (selectanswer === "flee") {
+    showMessage("We'll flee to mars this december", 'user')
+    setTimeout(botReply4Flee, 1000)
+}
+  dropdown.disabled = true            //prevent user from chosing two options in a row in dropdown menu
+}
+
+const userReply3No = () => {
+  const dropdown = document.getElementById("select")
+  const selectanswer = dropdown.options[dropdown.selectedIndex].value;
+ 
+  if (selectanswer === 'positive') {
+    showMessage("Sure, that seems like fun", 'user')
+    setTimeout(botReply4positive, 1000)
+} else if (selectanswer === "why") {
+    showMessage("What's in it for you?", 'user')
+    setTimeout(botReply4why, 1000)
+}
+  dropdown.disabled = true
+}
+
+  
+//Step 4
+//declaring functions that triggers bot replies, depending on the dropdown option user has chosen
+const botReply4Quarantine = () => {
+  showMessage("Quarantine.. In outer space you are always in quarantine. Come join me on my planet Frontus!", 'bot')
+  exitChat()    //trigger last function
+}
+
+const botReply4Flee = () => {
+  showMessage("Mars!? Frontus is a much better planet. We have the best take out in the universe!", 'bot')
+  exitChat()
+}
+  
+const botReply4positive = () => {
+  showMessage("Pack your bags and I'll pick you up in my spaceship!", 'bot')
+  exitChat()
+}
+
+const botReply4why = () => {
+  showMessage("We could use humans to do some research on the human anatomy, haha", 'bot')
+  exitChat()
+}
+
+const exitChat = () => {  //last function with yes/no buttons
+
+  inputWrapper.innerHTML = `  
+  <button id='letsGo'>Let's go!</button>
+  <button id='leave'>Leave me alone!</button>
+  `
+
+  document.getElementById('letsGo').addEventListener('click', userReply4letsGo)
+  document.getElementById('leave').addEventListener('click', userReply4leave) 
+}
+
+const userReply4letsGo = () => {
+  showMessage("Let's go!", 'user')
+  setTimeout(rocket, 1000)  
+ } 
+
+const userReply4leave = () => {
+  showMessage("Leave me alone!", 'user')
+  setTimeout(exit, 1000) 
+} 
+
+//End screens
```

Comment:
The end screens were really nicely done, looked really good! Hated the scary one!